src/etc/inc/interfaces.inc

```diff
@@ -6000,12 +6005,11 @@ function get_interface_vendor_mac($interface) {
 	global $g;
 
 	$mac = "";
-	$mac_file = "{$g['cf_conf_path']}/vendor_mac_{$interface}";
+	$mac_file = "{$g['vardb_path']}/vendor_mac";
 	if (file_exists($mac_file)) {
-		$mac = file_get_contents($mac_file);
-		if (!is_macaddr($mac)) {
-			$mac = "";
-		}
+		$vendor_mac_contents = file_get_contents($mac_file);
+		$vendor_mac_arr = json_decode($vendor_mac_contents, true);
```

Comment:
It's not necessary to set a variable for a single use, you can call json_decode direct on file_get_contents();